modules/AlphaWalletFoundation/AlphaWalletFoundation/EtherClient/EIP712TypedData.swift

```diff
@@ -35,6 +35,34 @@ public struct EIP712TypedData: Codable {
         }
     }
 
+    public var server: RPCServer? {
+        switch domain {
+        case .object(let dictionary):
+            switch dictionary["chainId"] {
+            case .string(let str):
+                if let server = eip155URLCoder.decodeRPC(from: str) {
```

Comment:
I think it shouldn't ever be be EIP155 format? Just `"123"` or `123`? (it probably shouldn't be `"123"` too, but we can loose a little I guess.

EIP-712 (https://eips.ethereum.org/EIPS/eip-712) says:

> uint256 chainId the [EIP-155](https://eips.ethereum.org/EIPS/eip-155) chain id. The user-agent should refuse signing if it does not match the currently active chain.